.travis.yml

```diff
@@ -14,13 +14,17 @@ os:
 env:
 - ARCH=x86_64
 
+before_install:
+- nvm install 8.10.0
```

Comment:
Can we not use `nvm use stable`  any particular reason for `8.10.0` ?

Reply:
There is no particular reason, will try with `nvm use stable`